01. 기본문제 풀이/07. 10부제/김보경/index.js

```diff
@@ -1,6 +1,18 @@
 function solution(day, arr) {
   let answer = 0;
 
+  arr.forEach((el) => {
+    let add = el.toString().split("");
+    add[1] == day ? answer++ : answer;
```

Comment:
동등비교연산자는 지양해야한다고 생각합니다. 
그래서 add[1] 형변환을 시켜주는 것이 낫지 않을까,,?라고 생각합니다
```suggestion
    +add[1] === day ? answer++ : answer;
```